tenets/codelingo/cockroachdb/fmt-verbs/codelingo.yaml

```diff
@@ -0,0 +1,89 @@
+funcs:
+  - name: hasGeneralVerb
+    type: asserter
+    body: |
+      function (str, order) {
+        order--
+        var matches = str.match(/%[A-Za-z]/g)
+        if (matches.length <= order) return false
+        var result = matches[order]
+        return result === "%v"
+      }
+  - name: verbForType
+    type: resolver
+    body: |
+      function (type) {
+        var types = {
+        "string":     "%s", 
+        "bool":       "%t",
+        "int":        "%d",
+        "int8":       "%d",
+        "int16":      "%d",
+        "int32":      "%d",
+        "int64":      "%d",
+        "uint":       "%d",
+        "uint8":      "%d",
+        "uint16":     "%d",
+        "uint32":     "%d",
+        "uint64":     "%d",
+        "uintptr":    "%p",
+        "byte":       "%d",
+        "rune":       "%q",
+        "float32":    "%g", 
+        "float64":    "%g",
+        "complex32":  "%g",
+        "complex64":  "%g",
+        }
+
+        var result = types[type]
+
+        return typeof result == 'undefined' ? "" : result
+      }
+tenets:
+  - name: fmt-verbs
+    vars:
+      result: "{{verbForType(argType)}}"
+    flows:
+      codelingo/docs:
+        title: Use Appropriate Verbs in Format
+        body: |
+          Prefer the most specific verb for your use. In other words, 
+          prefer to avoid %v when possible. However, %v is to be used 
+          when formatting bindings which might be nil and which do not 
+          already handle nil formatting. Notably, nil errors formatted 
+          as %s will render as "%!s()" while nil errors formatted as %v 
+          will render as "". Therefore, prefer %v when formatting errors 
+          which are not known to be non-nil.
+      codelingo/review:
+        comment:  type should use an appropriate verb such as `{{result}}` for `{{argType}}`
```

Comment:
getting MISSING for {{result}}

https://github.com/mullikine/mullikine.github.io/blob/master/codelingo/troubleshooting/tenets/fmt-verbs_getting-missing-for-comment-var.org

Reply:
Yeah, forgot about that MISSING thing. It's a go thing so somewhere along the way the "%d" string is being interpreted as a verb. I'll look into a way to force it to be treated as a plain string